src/runtimes/wapc.rs

```diff
@@ -129,8 +129,20 @@ pub(crate) fn host_callback(
                 "v1/is_certificate_trusted" => {
                     let req: CertificateVerificationRequest =
                         serde_json::from_slice(payload.to_vec().as_ref())?;
-                    let response = CertificateVerificationResponse {
-                        trusted: verify_certificate(req)?,
+                    let response: CertificateVerificationResponse = match verify_certificate(req) {
+                        Ok(b) => match b {
+                            BoolWithReason::True => CertificateVerificationResponse {
+                                trusted: true,
+                                reason: "".to_string(),
+                            },
+                            BoolWithReason::False(reason) => CertificateVerificationResponse {
+                                trusted: false,
+                                reason,
+                            },
+                        },
```

Comment:
nitpicking, this could be the implementation of `TryFrom`

Reply:
true! done in https://github.com/kubewarden/policy-evaluator/pull/214/commits/d21868bc1b20db0dc2f763c24f5e39e549aa6b01.

Reply:
The implementation of the From ends being [for CertificateVerificationResponse, here](https://github.com/kubewarden/policy-sdk-rust/pull/72/commits/a52b07c69023ba7f6f960b39a7076e9bfcd2fcbe).